ark/phenotyping/som_utils.py

```diff
@@ -166,6 +171,75 @@ def compute_cell_cluster_avg(cluster_path, column_prefix, cluster_col):
     return mean_count_totals
 
 
+def compute_cell_cluster_channel_avg(pixel_channel_avg, cell_counts,
```

Comment:
What is the point of doing the calculation this way? Why take a weighted average of an average, rather than just calculating the expression of each marker in each cell? Is this faster?